cfgov/v1/management/commands/search_pagerevisions.py

```diff
@@ -0,0 +1,60 @@
+import csv
+
+from django.core.management.base import BaseCommand
+from django.urls import reverse
+
+from wagtail.core.models import Page
+from wagtail.search.backends import get_search_backend
+
+from v1.models import IndexedPageRevision
+
+
+class Command(BaseCommand):
+    help = 'Search for a particular string within page revisions'
```

Comment:
It might be more helpful to say "search for words" within page revisions. A string might imply that you can search for partial words, or that an entire phrase is being searched for. Results I see return cases where any of the words entered appear, whether they're found together or not.